content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,28 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.12 [2020-06-12]
+
+### Features
+
+- Make all `pkg` applications stateful through stacks.
+- Add ability to export a stack's existing resource state as a `pkg`.
+- Update influx `pkg` commands with improved usage and examples in long form.
+- Extend `influx` CLI with version and User-Agent.
```

Comment:
..to include the version and User-Agent header with all http requests?

Reply:
the version is a new command, `influx version`; and the User-Agent header was also added. They landed in the same PR together.